content/style_guide.md

```diff
@@ -462,7 +484,7 @@ For example:
       {{</* /foundation_tabs_panel */>}}
     {{</* /foundation_tabs_panels */>}}
 
-#### Parameters
+#### Panel Parameters
```

Comment:
We should probably change "Parameters" on line 439 to "Tab Parameters" to match this.